test/wanda/checks_customizations_test.exs

```diff
@@ -268,4 +270,90 @@ defmodule Wanda.ChecksCustomizationsTest do
              ] = ChecksCustomizations.get_customizations(group_id)
     end
   end
+
+  describe "resetting customization" do
+    test "should return an error when attempting to reset a not existent customization" do
+      group_id = Faker.UUID.v4()
+
+      %{check_id: check_id} =
+        insert(:check_customization,
+          group_id: group_id
+        )
+
+      insert(:check_customization,
+        group_id: group_id
+      )
+
+      scenarios = [
+        %{
+          name: "non existent group id",
+          check_id: check_id,
+          group_id: Faker.UUID.v4()
+        },
+        %{
+          name: "non existent check id",
+          check_id: Faker.UUID.v4(),
+          group_id: group_id
+        },
+        %{
+          name: "non existent group and check id",
+          check_id: Faker.UUID.v4(),
+          group_id: Faker.UUID.v4()
+        }
+      ]
+
+      for %{check_id: possibly_invalid_check_id, group_id: possibly_invalid_group_id} <- scenarios do
+        assert {:error, :customization_not_found} =
+                 ChecksCustomizations.reset_customization(
+                   possibly_invalid_check_id,
+                   possibly_invalid_group_id
+                 )
+      end
+
+      assert CheckCustomization
+             |> Repo.all()
+             |> length() == 2
+    end
+
+    test "should reset customizations for a check" do
+      group_id = Faker.UUID.v4()
+
+      %{check_id: check_id_1} =
+        insert(:check_customization,
+          group_id: group_id
+        )
+
+      %{check_id: check_id_2} =
+        insert(:check_customization,
+          group_id: group_id
+        )
+
+      load_customizations = fn group_id ->
+        Repo.all(
+          from c in CheckCustomization,
+            where: c.group_id == ^group_id
+        )
+      end
+
+      assert [
```

Comment:
question: why do we need to assert this? we are 100% sure this is true, aren't we?
if not, we might have some flakiness in the test

Reply:
Yeah, not strictly necessary indeed.
It was mainly a way to show the _before_ and _after_ reset.

I don't mind removing the extra assertion 0cdb1da4c5a419f0f18bc0775504aba35ffb9f94